.github/workflows/unit-tests-workflow-for-customImportMap.yml

```diff
@@ -2,21 +2,32 @@ name: Unit tests workflow
 on:
```

Comment:
In my understand, this workflow only related to customImportMap plugin, which plugin is under `src/plugins/custom_import_map` path, can we also specify a path condition check? In that case, other maps plugins change under `src/plugins/` won't trigger this workflow, the code could be:
```
on:
  push:
    branches:
      - "*"
      - "feature/**"
    paths:
      - src/plugins/custom_import_map/**
  pull_request:
    branches:
      - "*"
      - "feature/**"
    paths:
      - src/plugins/custom_import_map/**
```

Reply:
Other plugins won't trigger this workflow because they are in their own repo.

Reply:
Talked with @junqiu-lei offline. We are going to add another plugin in this repo. https://github.com/opensearch-project/dashboards-maps/tree/feature/new-maps/src/plugins/maps_dashboards

So, restricting the github action to the specific folder.

Reply:
There have another developing maps visualization plugin in this repo https://github.com/opensearch-project/dashboards-maps/tree/feature/new-maps/src/plugins/maps_dashboards. Just also offline synced with Heemin.